triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/dao/DbTriggerDao.scala

```diff
@@ -153,7 +153,8 @@ final class DbTriggerDao private (dataSource: DataSource with Closeable, xa: Con
       pkgPayload: Array[Byte]): Either[String, (PackageId, DamlLf.ArchivePayload)] =
     for {
       pkgId <- PackageId.fromString(pkgIdString)
-      payload <- Try(DamlLf.ArchivePayload.parseFrom(pkgPayload)) match {
+      cos = Reader.damlLfCodedInputStreamFromBytes(pkgPayload)
+      payload <- Try(DamlLf.ArchivePayload.parseFrom(cos)) match {
         case Failure(err) => Left(s"Failed to parse package with id $pkgId.\n" ++ err.toString)
```

Comment:
Not sure what `cos` stands for. Maybe a more descriptive name?

Reply:
coded stream, it’s what we use in all other places. I agree that it’s a bit confusing. Does it make sense to just change it here or is that just making it even more confusing?

Reply:
Fair enough, if it's used in other places it makes sense to be consistent.